packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -72,7 +71,8 @@ export const colorPickerFactory = <T extends ColorPickerProps>(
                     ref={this.pickerTriggerRef}
                     onClick={showPopper}
                     onMouseLeave={hidePopper}
-                    color={theme.visualization.getColorByName(this.props.color || '#000000')}
+                    color={theme.visualization.getColorByName(color || '#000000')}
+                    aria-label={color}
```

Comment:
Are there cases where these two should differ, or could you assign them to the same variable?

Reply:
The color that is passed to the ColorSwatch is actually a hex color and we want the aria-label to have the name of the color if there is one. That's why we have it like this.